target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -811,32 +811,10 @@ static void rtl93xx_phylink_mac_config(struct dsa_switch *ds, int port,
 
 	sds_num = priv->ports[port].sds_num;
 	pr_info("%s SDS is %d\n", __func__, sds_num);
-	if (sds_num >= 0) {
-		switch (state->interface) {
-		case PHY_INTERFACE_MODE_HSGMII:
-			sds_mode = 0x12;
-			break;
-		case PHY_INTERFACE_MODE_1000BASEX:
-			sds_mode = 0x04;
-			break;
-		case PHY_INTERFACE_MODE_XGMII:
-			sds_mode = 0x10;
-			break;
-		case PHY_INTERFACE_MODE_10GBASER:
-		case PHY_INTERFACE_MODE_10GKR:
-			sds_mode = 0x1b; /* 10G 1000X Auto */
-			break;
-		case PHY_INTERFACE_MODE_USXGMII:
-			sds_mode = 0x0d;
-			break;
```

Comment:
Were the dropped modes secretly not supported?

Reply:
From what I can tell `sds_mode`  was set but then never used here. Threw it out since I was touching the code anyways to handle 1G modes. Am I missing something here?

Reply:
OK, didn't realize it was dead code. Could split it off into a separate commit? Then it would be more clear why the code is being removed, and make it easier to find it back if someone ever want to resurrect it.

Reply:
Will do. Just giving this whole MR an overhaul since I finally found some more time for RTL39xx stuff